src/Domain/Creators/BinaryCompactObject.hpp

```diff
@@ -316,6 +318,15 @@ class BinaryCompactObject : public DomainCreator<3> {
     static constexpr Options::String help = {"Radius of the entire domain."};
   };
 
+  struct OpeningAngle {
```

Comment:
Same comment here about clarifying what "opening angle" means for a full envelope of wedges. Maybe `HalfWedgesOpeningAngle` if you want to retain the ability to choose any value for the angles. Values of `45` and `60` degrees would correspond to the previous behavior and the new equal angular size behavior, respectively (note that in the help string). Alternatively, just a boolean like `EqualizeOpeningAngles` or something like that could also work. WDYT?

Reply:
(as mentioned above I think it's easier to be able to compare two half wedges with the end cap wedge using the full opening angle vs the half)

I'm not sure if we want to equalize by default yet, since we are sacrificing equiangular-ity in the endcap wedges (but so far runs have not shown poor behavior in those regions resulting from the change).

I have a guess that a value between 120.0 and 90.0 might be ideal to strike a balance between angular size and equiangular-ity, or it could also be the case that we don't want to equalize angular size in angle but rather in surface area / volume covered by each wedge/half wedge - it is not clear to me that a choice of 120.0 results in the half wedge having the same surface area or volume as an endcap wedge (and this still might not be what gives ideal behavior)

In any case, I think it's too soon to default yet :sweat_smile: